test/common/grpc/context_impl_test.cc

```diff
@@ -87,5 +88,28 @@ TEST(GrpcContextTest, ResolveServiceAndMethod) {
   EXPECT_FALSE(context.resolveDynamicServiceAndMethod(path));
 }
 
+TEST(GrpcContextTest, ResolveServiceAndMethodNotChanged) {
```

Comment:
Nit: The name of the test case feels a bit misleading as the names of the request are actually changed.

Reply:
Does this become clearer now?